prac_09/sort_files_v2.py

```diff
@@ -0,0 +1,36 @@
+"""
+CP1404 Practical
+File moving based on user input exercise
+"""
+import shutil
+import os
+
+
+def main():
+    """Move files with the os module."""
+    # change to desired directory
+    os.chdir('FilesToSort')
+    extention_dict = {}
+    for filename in os.listdir('.'):
+        if not os.path.isdir(filename):
+            file_ext = os.path.splitext(filename)[1].replace(".", "")
```

Comment:
Why? Won't all extensions start with a dot? Then you can just use as is. Or, you could use slicing to remove the first character if you preferred.